templates/not-found.html

```diff
@@ -2,8 +2,19 @@
 
 {% block title %}
     <title>WhoTracks.me | Page Not Found </title>
+    <meta name="description" content="The page you are trying to visit does not exist. We have lots of other interesting things for you to see though, visit https://whotracks.me to find out.">
 {% endblock %}
 
+{% block og_params %}
+    <!-- OG params for sharable content -->
+    <meta property="og:title" content="WhoTracks.me | Page Not Found">    
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://whotracks.me/not-found.html">
+    <meta property="og:description" content="The page you are trying to visit does not exist. We have lots of other interesting things for you to see though, visit https://whotracks.me to find out.">
```

Comment:
Do we need open graph data for 404s? Hopefully people won't be sharing them...

Reply:
Inexpensive. Just in case of sharing outdated/bad links.